docs/development.md

```diff
@@ -0,0 +1,24 @@
+```{include} ../CONTRIBUTING.md
+```
+
+## Contributing documentation
+
+### Building the documentation locally
+
+To build the documentation locally, make sure you install the necessary dependencies. You can do this by navigating to your local clone of the MaxText repo and running:
+
+```bash
+pip install -r requirements_docs.txt
+```
+
+Once the dependencies are installed, you can navigate to the `docs/` folder and run:
+
+```bash
+sphinx-build -b html . _build/html
+```
+
+This will generate the documentation in the `docs/_build/html` directory.
```

Comment:
Could you please clarify if this is needed for documentation deployment, or optional just for locally pre-viewing / testing the docs ?

Reply:
Let me know if this updated version is better.

Reply:
Added a suggestion to make this a bit clearer.

Reply:
Also, what about adding a note about how folks can check the built `html` either by opening these directly or using something like `python -m http.server -d docs/_build/html` ?